src/Common/Commands.Common/CustomAttributes/BreakingChangeAttributeHelper.cs

```diff
@@ -77,7 +77,7 @@ public class BreakingChangeAttributeHelper
          * the boundParameterNames is a list of parameters bound to the cmdlet at runtime, 
          * We only process the Parameter beaking change attributes attached only params listed in this list (if present)
          * */
-        public static void ProcessCustomAttributesAtRuntime(Type type, InvocationInfo invocationInfo)
+        public static void ProcessCustomAttributesAtRuntime(Type type, InvocationInfo invocationInfo, Action<string> writeOutput)
```

Comment:
I would call this 'writeMessage'  WriteOutput is already the name of the PS method to write to the output stream, which are the values returned from the cmdlet, rather than messages.  Best to ensure there is no confusion.